Mathlib/RingTheory/Ideal/Quotient/Defs.lean

```diff
@@ -63,8 +63,9 @@ protected def ringCon (I : Ideal R) : RingCon R :=
         rw [mul_sub, sub_mul, sub_add_sub_cancel, mul_comm, mul_comm b₁]
       rwa [← this] at F }
 
-instance commRing (I : Ideal R) : CommRing (R ⧸ I) :=
-    inferInstanceAs (CommRing (Quotient.ringCon I).Quotient)
+instance commRing (I : Ideal R) : CommRing (R ⧸ I) where
```

Comment:
```suggestion
-- This instance makes use of the existing AddCommGroup instance to boost performance.
instance commRing (I : Ideal R) : CommRing (R ⧸ I) where
```

Reply:
This modification to [switch to the `ring` instance](https://github.com/leanprover-community/mathlib4/blob/77fe4cd4369879746c0657d048678bc8090b6aef/Mathlib/RingTheory/Ideal/Quotient/Defs.lean#L52-L57) in #17930, but I think there's no harm getting it merged now.